replica/database.hh

```diff
@@ -562,10 +562,8 @@ private:
         return _config.enable_cache && _schema->caching_options().enabled();
     }
     void update_stats_for_new_sstable(const sstables::shared_sstable& sst) noexcept;
+    void update_sstables_stats(ssize_t count, int64_t bytes_on_disk) noexcept;
     future<> do_add_sstable_and_update_cache(sstables::shared_sstable sst, sstables::offstrategy offstrategy);
```

Comment:
it seems we now have the potential to calcualte these stats on the fly using sstables_set, therefore avoiding this boring independent tracking.